Makefile

```diff
@@ -49,8 +49,21 @@ GCOVRURL=git+https://github.com/nschum/gcovr.git@never-executed-branches
 VERSION=$(shell ./setup.py version | grep Version | awk '{print $$4}' \
 	| sed 's/+/-/')
 
+DEFINES += -DNDEBUG -DVERSION=$(VERSION) -DSEQAN_HAS_BZIP2=1 \
+	   -DSEQAN_HAS_ZLIB=1 -UNO_UNIQUE_RC
+
 INCLUDESTRING=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
 	    | grep '^ /' | grep -v cc1plus)
+INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
+	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
+PYINCLUDE=$(shell python2.7-config --includes)
```

Comment:
We should switch to something that works with python as well as legacy python. While looking into this I also realised that I don't know how to make this tell me the headers that match the python version in my virtual/conda env